### ISSUE TITLE
Original ethereum address attached to account 1 is different after carrying out account recovery by seedphrase? Lost access to my original address and token balances now showing 0  as wallet is controlling new empty address :(

### ISSUE THREAD

**Issue by BGzetro** [MetaMask/metamask-extension#3258]
This problem occured about 8 days ago, afew days after creating my first metamask wallet (using the firefox extension). I installed the extension on both firefox and google chrome. I used it mainly on google chrome and on my second attempt of using it on google chrome it would not let me log in with my correct password. I tried on firefox and same problem. I decided to go through the recovery process (through the google chrome extension) to create a new password and used my 12 word seed phrase to restore my wallet (I had this backed up in to places in encrypted files).

Once my wallet had restored I could see my wallet was showing 0 balance for all my tokens although it still remembered the names of the tokens that were previously tied to my account, just 0 balance showing for them. I compared the address showing for account 1 and noticed it was completely different to my original address for account 1. I checked my original address on etherscan (the one i no longer have access to) and could see all my tokens still safe.

I did not record my private key before this fault occurred, I only recorded my seedphase, password and ethereum address. After the fault occurred i recorded my private key and I assume this is linked to my original seed phase so should be the same before and after the fault as my seed phase is still the same, is that correct?

A couple of days before the fault occured I bought some PRL off Kucoin and transferred them to my wallet. A few hours later I used etherdelta to buy REAL and transferred them to my wallet also (I connected to etherdelta using metamask). 5 days later etherdelta was down (still running but not functioning correctly) and is still down. Could etherdelta have corrupted my account for when i was to recover it a couple of days later perhaps?? I'm not sure :(

For both Firefox and google chrome I use Metamask by Danfinlay, Kumavis ver 3.14.1

It's alot of money to me, i'm pretty devestated and will take me a very long time to make it back, I only tried metamask as it was suppose to be safer than using myetherwallet where you paste your private key to gain access to your wallet, guess that information wasn't 100% correct :(

The address my metamask wallet gives me for account 1 now is:
0x83b5661a810ef02990926a9ec6310a5b9ef087c0

Before I restored from seed phrase it was:
0xbc70688f0394d98C6016F670d2E2515D0Ef63533

I only ever created 1 account with my wallet, since going through further recovery processes i've added additional accounts, hoping one will show my original address but no luck.

I've also used the process where you use myetherwallet to check all the addresses linked to your 12 word seed phase, i've been scrolling through and no match. 

**If someone provides information or help that leads to me gaining access to my original address through my wallet, I'll transfer 15% of all my tokens to you which is today eqivalent to $1230 as of 15/02/18.
In 10 years i estimate it will worth $80,000 or many multiples of that. If multiple people lead to me gaining access again then i'll increase what I give to 20% but will split it between those that really helped by how much they helped.**

**Comment by developerhub0** [comment]
same with me.....

**Comment by BGzetro** [comment]
This problem occured about 8 days ago, afew days after creating my first metamask wallet (using the firefox extension). I installed the extension on both firefox and google chrome. I used it mainly on google chrome and on my second attempt of using it on google chrome it would not let me log in with my correct password. I tried on firefox and same problem. I decided to go through the recovery process (through the google chrome extension) to create a new password and used my 12 word seed phrase to restore my wallet (I had this backed up in to places in encrypted files).

Once my wallet had restored I could see my wallet was showing 0 balance for all my tokens although it still remembered  the names of the tokens that were previously tied to my account, just 0 balance showing for them.  I compared the address showing for account 1 and noticed it was completely different to my original address for account 1. I checked my original address on etherscan (the one i no longer have access to) and could see all my tokens still safe.

I did not record my private key before this fault occured, I only recorded my seedphase, password and ethereum address.

After the fault occurred i recorded my private key and I assume this is linked to my original seed phase so should be the same before and after the fault as my seedphase is still the same, is that correct?

A couple of days before the fault occured I bought some PRL off Kucoin and transfered it to my wallet. A few hours later I used etherdelta to my REAL and transfered them to my wallet also (I connected to etherdelta using metamask). 5 days later etherdelta was down (still running but not functioning correctly) and is still down, could etherdelta have corrupted my account for when i was to recover it perhaps?? I'm not sure :(

For both Firefox and google chrome I use Metamask by Danfinlay, Kumavis ver 3.14.1

It's alot of money to me, i'm pretty devestated and will take me a very long time to make it back, I only tried metamask as it was suppose to be safer than using myetherwallet where you paste your private key to gain access to your wallet, guess that information wasn't 100% correct :( 

The address my metamask wallet gives me for account 1 now is:
0x83b5661a810ef02990926a9ec6310a5b9ef087c0

Before I restored it was:
0xbc70688f0394d98C6016F670d2E2515D0Ef63533

I only ever created 1 account with my wallet,  since going through further recovery processes i've added additional accounts, hoping one will show my original address but no luck. 


If someone provides information or help that leads to me gaining access to my original address through my wallet, I'll transfer 15% of all my tokens which is today eqivalent to $1230 as of 15/02/18. 
In 10 years i estimate it will worth $80,000 or many multiples of that. If multiple people lead to me gaining access again then i'll increase what I give to 20% but will split i'll between those that really helped by how muched they helped.

**Comment by BGzetro** [comment]
developerhub0 when did the problem happen to you?

**Comment by developerhub0** [comment]
yes. I deleted my browser cache and my account got deleted. I tried to restore my account with my recovery seeds. My account got restored but with different address with 0 balance....

**Comment by BGzetro** [comment]
I deleted my browser history in between those days but don't think i did with my cache. 
Also how long ago was it when this problem happened to you? Did you manage to re-gain access to your account in the end?

**Comment by BGzetro** [comment]
Same here I wrote to them last week and have had to replies trying to help out but no success yet.
They sent me a link to an issue they had about a year ago which sounds the same but still no solution to it yet:

https://medium.com/metamask/metamask-security-advisory-and-bug-bounty-for-seed-phrase-concern-bbd95ab63210

**Comment by tmashuang** [comment]
A couple insights about seed phrases are, they are very precise and adding an extra prefix/suffix to a word can still be a valid seed phrase to generate an list of accounts. Initially, if the original seed phrase was written incorrectly it may generate a new list of accounts. This issue has been an ongoing investigation to better improve first account creation. We have the bounty out for any external contributors to investigate as well and another issue thread for it, #3127

**Comment by BGzetro** [comment]
I understand that about the seed phrase which is why I copied and pasted it it twice to 2 different encrypted files before saving. They are both the same so no error with the seed phrase on my side it seems.

**Comment by tsunamitreats** [comment]
This just happened to me.  I think firefox updated itself and Metamask disappeared on me.  I can't recover my account and the couple accounts I can recover, aren't the address I expect.  I'm restoring from JSON file.

I can see that my tokens are in tact in the address I expect, but I now don't have access to this account via metamask...

**Comment by tmashuang** [comment]
@tsunamitreats If you created an account and exported a JSON/UTC file from MEW/MyCrypto/Mist, and imported it in MetaMask, they do not associate to the MetaMask seed phrase. https://metamask.helpscoutdocs.com/article/17-what-are-loose-accounts

**Comment by BGzetro** [comment]
Hi,


I did not import a JSON/UTC file from MEW/MyCrypto/Mist to Metamask to restore my account.


I was thinking it might be more of an issue relating to this:

https://github.com/MetaMask/metamask-extension/issues/2577



It was original classed as fixed at the end of November last year but i've got a feeling it is still at fault.


Something along what i write below occurred in the past and it seems like this may still be an issue:


  *   When an account was first created a user would click on 'create vault' but it was selected multiple times.

  *   Multiple vaults are then created but only the seed for the first vault is shown immediately and generation of the other vaults are stalled for some reason.

  *   The seed phase continues to show the first vault seed phrase despite the subsequent vaults generation completing.

  *   The last vault to finish the generation process overwrites the first vault.

  *   The seed recorded is for the first vault which had been overwritten, this in unknown to the user. The user sends tokens to their vault which their seed doesn't relate to and on account restore the seed they recorded instead restores the 1st vault, the one which had been overwritten and didn't have any tokens stored on. The user loses access to the correct vault as they were never shown the correct seed phrase on account creation.


[https://avatars0.githubusercontent.com/u/542863?s=400&v=4]<https://github.com/MetaMask/metamask-extension/issues/2577>

Reproduce a bug where backed-up seed phrase does ... - GitHub<https://github.com/MetaMask/metamask-extension/issues/2577>
github.com
Join GitHub today. GitHub is home to over 20 million developers working together to host and review code, manage projects, and build software together.




________________________________

**Comment by ELREYDELKAOS** [comment]
Hola, pudiste solucionarlo? a veces funciona el instalar una versión antigua de Metamask. Hay otras opciones que podemos hablar si quieres.

**Comment by andriytyurnikov** [comment]
@BGzetro reported account 0xbc70688f0394d98C6016F670d2E2515D0Ef63533 has zero balance now

**Comment by BGzetro** [comment]
I managed to withdraw my funds after finding a 12 word seed phase I used when I first created a different meta mask account years ago. That 12 word seed phase was able to recover my funds by showing the correct addresses again.

What I worked out occured.

When I created a new account years after having another account (which I stopped using), i was shown a new 12 word seed phase which was suppose to be able to recover the funds of my new addresses.
When I went to recover my funds of my new address with the new 12 word seed phase, totally different addresses were shown with no balance.
When I created my second meta mask account it seems that the generated addresses were that of my old account but I was shown a 12 word seed phase that was to be used with a new set of addresses that was never showed in my account, instead my old address tied to my old 12 word seed phrase was shown. 

I believe this bug still has not been fixed but luckily I had my 12 word seed for my old account which I did not use still so I could recover and withdraw my funds.

**Comment by andriytyurnikov** [comment]
@BGzetro , it sounds like you've entered wrong 12 words

**Comment by BGzetro** [comment]
@andriytyurnikov it would seem like that but I can assure you that was not the case.

When you create a new  account, 12 word seed phase shown should recover the funds  of the addresses shown in that account. 

I used that 12 word seed phase and it did not recover the funds of the address original shown in that account, it just brought up addresses I had never seen as the seed phase I was shown must not have been tied to the addresses I deposited my funds to. The seed phrase from the account I used years ago was. 
Both accounts were created on the same PC and used Mozilla Firefox and Google Chrome browser with both and had re-installed both browsers at least once before I had created my second meta mask account.

**Comment by BGzetro** [comment]
Maybe using more than one browser for different accounts and re-installed them created the issue. Im not sure

**Comment by andriytyurnikov** [comment]
@BGzetro I quote

>  I managed to withdraw my funds after finding a 12 word seed phase I used when I first created a different meta mask account years ago. 

Which obviously means that account you was looking for was accessible by "another 12 words"

**Comment by BGzetro** [comment]
@andriytyurnikov
I do waffle a bit lol. 
Yes your summary is correct.

The problem should not occur though.

**Comment by pasamusa** [comment]
Same thing happened to me. The 12 words take me to a different wallet address with zero balance. Have been 4 month of going back and forward with the customer service, but they try any excuse to tell me that the frase I am using is not the correct one, when it was given to me by Exudus before I transfer some coins. Had to reinstall windows and Exodus. When I recover the account with the 12 words, my balance was 0 and the ETH address was different. I check in ethplorer and the old wallet still have the crypto. How can I access that account to retrieve my money?  Thanks in advance, Pasam

**Comment by zanja85** [comment]
I had a simmilar issue today. I was importing metamask on a new computer, and was assigned a completely new address with no funds on. So I tried again in another browser and got the same result. Then I tried to import my "old account" with the private key, and got some weird error message.

Eventually I found out that when I was copying seedword/private key there was an extra blanspace after the last word/letter that was subitted. When I again pasted my private key and then hit the backspace key once before pressing "okay" it worked out fine.

**Comment by cultismael** [comment]
Stolen Bitcoins Recovered 

My Cash App was hacked and all my Cash and Bitcoins was stolen , but after I reported to Mrs Debra Sabina, She helped me applied for a compensation and today I am happy and able to meet up with my needs.

if anyone of you have lost money online or bitcoins and any other valuable crypto currencies, kindly contact Mrs Debra Sabina now to help you recover it all back or help you get a compensation.

you can also set up good bitcoin investments with Mrs Debra Sabina

Contact Mrs Debra Sabina via Text/whatsapp +14132170251 or via email : debrasabina1961@gmail.com

Thank You

**Comment by suzannebalson4-gif** [comment]
In early 2025, I was living in the vibrant heart of Austin, Texas, surrounded by a booming tech scene that had sparked my interest in cryptocurrency. As a software engineer who had worked diligently for years, I had managed to save $65,700 through my salary and a lucrative side venture developing mobile apps. Eager to secure my financial future, I began exploring investment opportunities in the crypto market with great enthusiasm. One day, while browsing online, I stumbled upon what appeared to be a legitimate investment platform promising extraordinary returns. The website was impeccably designed, and the testimonials from supposed satisfied investors looked authentic. Fueled by confidence, I made the impulsive decision to invest my entire life savings, convinced I was making a prudent financial move. However, my exhilaration quickly turned into despair when I attempted to withdraw my initial investment. The website became unresponsive, and my emails to customer support went unanswered. It soon became painfully clear that I had fallen victim to a sophisticated scam. Desperate to reclaim my funds, I embarked on a frantic search to identify the scammers and locate the elusive website. In a moment of regrettable desperation, I invested an additional $20,000 in Bitcoin, hoping to hire a private investigator to help trace the criminals. Unfortunately, this only led to further frustration and loss, as I realized I was entangled with a well-organized criminal enterprise that had expertly concealed its tracks. Feeling utterly defeated and hopeless, I nearly gave up. That was until I stumbled upon an article extolling the virtues of DIGITAL LIGHT SOLUTION Web https://digitallightsolution.com/. Skeptical but having nothing left to lose, I reached out to their team. Unlike the silence I had experienced before, they responded immediately with empathy and a clear plan of action. Their experts utilized advanced forensic tools to trace the flow of my stolen funds, navigating the complex blockchain trails that the scammers thought were invisible. Within weeks, Digital Light Solution not only identified the network behind the fraud but also successfully facilitated the recovery of the majority of my lost capital, including my original savings and the additional funds I had foolishly poured into the investigation. They guided me through every step with transparency and professionalism, restoring not just my finances but my peace of mind. Today, I am back on my feet, wiser and more cautious, but most importantly, grateful. If you find yourself in the dark nightmare of a crypto scam, do not lose hope. Digital Light Solution was the beacon that guided me out of the darkness and back to safety. Contact them directly Email —D.i.g.i.t.a.l.l.i.g.h.t.s.o.l.u.t.i.o.n.@.q.u.a.l.i.t.y.s.e.r.v.i.c.e.(DOT).c.o.m And Telegram User ——D.i.g.i.t.a.l.l.i.g.h.t.s.o.l.u.t.i.o.n

![Image](https://github.com/user-attachments/assets/490aa491-9b4a-4fc7-81d7-b5e739529a5d)
